cni/ipam/plugin/main.go

```diff
@@ -24,16 +27,20 @@ var version string
 func main() {
 	var config common.PluginConfig
 	config.Version = version
-	logDirectory := "" // Sets the current location as log directory
 
-	log.SetName(name)
-	log.SetLevel(log.LevelInfo)
-	if err := log.SetTargetLogDirectory(log.TargetLogfile, logDirectory); err != nil {
+	loggerCfg := &log.Config{
+		Level:       zapcore.DebugLevel,
```

Comment:
what is this level indicate? I see in all place we use either logger.Info or logger.Error